### PR TITLE
[WIP] Add support of PowerVS on hypershift-hosted test

### PR DESCRIPTION
**Pull request by Neha-dot-Yadav**
Testing AWS Side

### CODE REVIEW

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-hypershift-hostedcluster-e2e

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse  release-openshift-origin-installer-launch-hypershift-hosted-powervs

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 1]
```diff
@@ -2877,3 +2877,116 @@ periodics:
...
+        value: "9000"
+      - name: BRANCH
+        value: "4.12"
+      - name: JOB_NAME_SAFE
+        value: launch
+      - name: NAMESPACE
+      - name: UNRESOLVED_CONFIG
+        value: |
+          base_images:
+          release:
```
Comment (dharaneeshvrd) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:2919]:
```suggestion
          releases:
```

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse release-openshift-origin-installer-launch-hypershift-hosted-powervs

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse release-openshift-origin-installer-launch-hypershift-hosted-powervs

**Comment by Neha-dot-Yadav** [comment]
/pj-rehearse release-openshift-origin-installer-launch-hypershift-hosted-powervs

**Pull request closed without merging**